lib/bolt/cli.rb

```diff
@@ -1056,7 +1056,7 @@ def config_loaded
       msg = <<~MSG.chomp
         Loaded configuration from: '#{config.config_files.join("', '")}'
       MSG
-      @logger.debug(msg)
+      @logger.info(msg)
```

Comment:
@beechtom Does this at least partially solve https://github.com/puppetlabs/bolt/issues/1995 ?

Reply:
Hm, I actually wonder if this belongs at the same level as `Loading modules from`. It seems like you'd only want to know this if Bolt wasn't working as expected, which seems like debug level? But it also seems like more of a pain point than users not having their modulepath set correctly, which maybe warrants a higher level. Either way it's not logged by default - I don't feel super strongly.

Reply:
I think ideally we would have info level logging about which project is being used so that if you're using a log file shared between projects, you can tell which project was being used. Then details about which config files, modulepath, inventory, etc could live at debug level. Since we don't have anything that is explicitly about the project, mentioning the config file at info level is a half measure.

Reply:
@lucywyman Pretty much what Nick said, but I do think logging this at info combined with it being written to the debug log by default is an improvement, since it will hopefully be a bit more discoverable. 🤞